CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Autokey no longer working with alacritty on X11
 - Freeze when moving window between monitors on Xfwm
 - Mouse cursor not changing on Wayland when cursor theme uses legacy cursor icon names
+- Deriving `ConfigDeserialize` takes the `#[config(alias = "...")]` attribute(s) into account
+- Compilation errors are correctly reported when deriving `ConfigDeserialize` macro
```

Comment:
We don't document things for developers, we document them for users.

So the message you should be adding is that config keys are now all available under their proper name.

Reply:
thank you. i did not know